### PR TITLE
fix(pml-toggle): Fix Toggle + misc

### PR DESCRIPTION
**Pull request by jaredvu**
- Fix `shouldHideLaunchableMarkets` toggle and convert from useState to redux state
- Give `AssetIcon` white background
- Fix position of `<AccountInfo>` component
- Fix tooltips that use `AssetId`

---

## Views

- `<PlaceOrderAndButtonReceipt>`, `<TradeSizeInputs>`
  - use `getDisplayableAssetFromBaseAsset` to fix display of tooltip

- `<MarketFilter>`
  - Fix toggle (it was inverted)
  - Use dispatch and selector for `shouldHideLaunchableMarkets`

- `<LaunchableMarket>`
  - keep UI refresh conditional consistent with `<Trade>` page 

## Components

- `<AssetIcon>`
  - give `img` a white background for icons that use transparency

## Constants/Types

- `constants/localStorage`
  - add `shouldHideLaunchableMarkets` localStorageKey

## Hooks

- `hooks/useMarketsData`
  - use redux version of `shouldHideLaunchableMarkets`

## State

- `state/configs`, `state/configsSelectors`
  - add `shouldHideLaunchableMarkets` to configs and selectors to persist user session

---

### CODE REVIEW

File: src/state/configs.ts [hunk 1]
```diff
@@ -132,6 +137,10 @@ export const configsSlice = createSlice({
         })
       );
     },
+    setShouldHideLaunchableMarkets: (state: ConfigsState, action: PayloadAction<boolean>) => {
```
Comment (tyleroooo) [inline src/state/configs.ts:140]: this is fine but why not use the new localstorage thing tina added?

File: src/views/MarketFilter.tsx [hunk 2]
```diff
@@ -46,36 +46,42 @@ export const MarketFilter = ({
...
   const navigate = useNavigate();
+  const dispatch = useAppDispatch();
   const { hasPotentialMarketsData } = usePotentialMarkets();
   const { uiRefresh, pml: showLaunchMarkets } = testFlags;
   const showProposeButton = hasPotentialMarketsData && !hideNewMarketButton;
+  const shouldHideLaunchableMarkets = useAppSelector(getShouldHideLaunchableMarkets);
+
+  const onShouldHideLaunchableMarkets = useCallback(
+    (shouldHide: boolean) => {
+      dispatch(setShouldHideLaunchableMarkets(!shouldHide));
```
Comment (tyleroooo) [inline src/views/MarketFilter.tsx:60]: does onCheckedChange really provide the old value?

**Review by tyleroooo — APPROVED** [review]

File: src/state/configs.ts (hunk 1 above)
Comment (jaredvu) [reply src/state/configs.ts:140]: redux-persist? I think Aleka is adding in her Favorites PR

File: src/views/MarketFilter.tsx (hunk 2 above)
Comment (jaredvu) [reply src/views/MarketFilter.tsx:60]: it does. its an `onChange` event so I have to invert in order for it to update

Comment (jaredvu) [reply src/views/MarketFilter.tsx:60]: Kind of wonky behavior honestly

File: src/state/configs.ts (hunk 1 above)
Comment (jaredvu) [reply src/state/configs.ts:140]: I made sure to add this to the config slice that will be added to persist later

File: src/views/MarketFilter.tsx (hunk 2 above)
Comment (tyleroooo) [reply src/views/MarketFilter.tsx:60]: surely this should just be `shouldHide` instead of `!shouldHide`

File: src/state/configs.ts (hunk 1 above)
Comment (tyleroooo) [reply src/state/configs.ts:140]: sounds good

File: src/views/MarketFilter.tsx (hunk 2 above)
Comment (tyleroooo) [reply src/views/MarketFilter.tsx:60]: oh I see

Comment (jaredvu) [reply src/views/MarketFilter.tsx:60]: If you use `shouldHide` then the redux dispatch will send in `false` since the state doesn't change

**Pull request merged**